cellx/tools/projection.py

```diff
@@ -33,9 +33,12 @@ class ManifoldProjection2D:
     Parameters
     ----------
     images : list
+        A list of image patches in np.ndarray format.
     output_shape : tuple
+        Final size of individual image patches in the projection space.
     preload_images: bool
-
+        Preload images or not. Preload given a list of image filenames. If not,
```

Comment:
If a list of filenames is provided in `image` this gives the option to preload them